### DISCUSSION TITLE
Error when running job render-page: Cannot find module

### DISCUSSION THREAD

**Discussion by FrostyMemes** [requarks/wiki#5462] [Error / Bug Report] [answered]
Hi, I added my markdown renderer to the server folder .\server\modules\rendering\. Named the new folder markdown-template-print, added definition.yml and render.js, all settings appeared in the rendering/markdown admin area, everything is trun on. But when I try to render the resulting text in the markdown editor window - wiki.js gives an error:

Error when running job render-page: Cannot find module '../markdown-template-printer/renderer.js'
Require stack:
- D:\Projects\wikijs\wikijs\wiki2\server\modules\rendering\markdown-core\renderer.js
- D:\Projects\wikijs\wikijs\wiki2\server\jobs\render-page.js
- D:\Projects\wikijs\wikijs\wiki2\server\core\worker.js

How to solve this problem?
![image](https://user-images.githubusercontent.com/49942326/178240330-0ba09d37-31f8-42d5-b6a8-3d155975eaa1.png)

**Comment by adonix-dev** [comment]
Same Issue .. any patch ?

**Reply by FrostyMemes** [reply, marked as answer]
> Same Issue .. any patch ?

My mistake was that I created the render.js file after building the project. The definition.yml and render.js file must be in the new render folder at the same time before building the project with the new folder.
